gfauto/gfauto/download_cts_gf_tests.py

```diff
@@ -35,8 +35,8 @@
 
 
```

Comment:
Worth a comment on what this function returns?

Reply:
I've been using a slightly weird convention where most functions typically return the "output_path" parameter. This allows you to do things like:

```python
foo_dir = f(a, b, c, output_dir=work / "foo")
bar_dir = g(foo_dir)
```

But still, there a lot of missing return comments!

Reply:
I added a coding conventions file. There are still a lot of missing docstrings, but I don't think I will fix this one in this PR.